helm_helpers.go

```diff
@@ -507,7 +507,7 @@ func deployTiller(namespace string, serviceAccount string, defaultServiceAccount
 	}
 	cmd := command{
 		Cmd:         "helm",
-		Args:        concat([]string{"init", "--force-upgrade"}, maxHistory, sa, tillerNameSpace, tls, storageBackend),
+		Args:        concat([]string{"init", "--upgrade"}, maxHistory, sa, tillerNameSpace, tls, storageBackend),
```

Comment:
I assume the intention is `helm init --upgrade --force-upgrade`, in which case the PR should just add `--upgrade` to the arg list.